docs/shared/guides/troubleshoot-convert-to-inferred.md

```diff
@@ -0,0 +1,89 @@
+# Troubleshoot Convert to Inferred Migration
+
+Nx comes with plugins that automatically [infer tasks](/concepts/inferred-tasks) (i.e.g Project Crystal) for your projects based on the configuration of different tools. Inference plugins comes with many benefits, such as reduced boilerplate, and access to features such as [task splitting](/ci/features/split-e2e-tasks). To make the transition easier for existing projects that are not yet using inference plugins, many plugins provide the `conver-to-inferred` generator that will switch from executor-based tasks to inferred tasks.
+
+To see a list of the available migration generator, run:
+
+```shell
+nx g conver-to-inferred
+```
+
+This will prompt to choose a plugin to run the migration for.
+
+Although the `convert-to-inferred` generator should work for most projects, there are situations that additional changes to be done by hand. If you run into issues that are not covered on this page, please open an issue on [GitHub](https://github.com/nrwl/nx/issues).
+
+## Error: The nx plugin did not find a project inside...
+
+This error occurs when configuration file matching the tooling cannot be found. For example, Vite works with `vite.config.ts` (or `.js`, `.cts`, `.mts`, etc.). If you've named your configuration file to something unconventional, you must rename it back to the standard naming convention before running the migration generator again.
+
+For example, if you have a `apps/demo/vite.custom.ts` file and are running `nx g @nx/vite:convert-to-inferred`, you must first rename the file to `apps/demo/vite.config.ts` before running the generator.
+
+## Vite: Unsupported `proxyConfig` Option
```

Comment:
These plugin specific errors should probably be located on the `convert-to-inferred` generator page for that plugin.  You could link to them from here.

Reply:
I think we can do a follow-up on this.